requirements/base.txt

```diff
@@ -67,11 +67,11 @@ jsonfield2==4.0.0.post0
     # via edx-celeryutils
 kombu==5.0.2
     # via celery
-newrelic==5.24.0.153
+newrelic==6.0.1.155
```

Comment:
same story, the major version bump appears to be due to dropping py3.5 support, about which we don't really need to be concerned https://docs.newrelic.com/docs/release-notes/agent-release-notes/python-release-notes/python-agent-600154/